src/Core/Migration/V6_5/Migration1687462843ProductManufacturerMediaThumbnails.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_5;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+/**
+ * @internal
+ */
+#[Package('core')]
+class Migration1687462843ProductManufacturerMediaThumbnails extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1687462843;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $thumbnailSizes = $this->upsertThumbnailSizes($connection);
+
+        $configurationId = $connection->fetchOne(
+            'SELECT media_folder_configuration_id FROM media_folder WHERE name = :name',
+            ['name' => 'Product Manufacturer Media']
+        );
+
+        if (!$configurationId) {
+            return;
+        }
+
+        foreach ($thumbnailSizes as $thumbnailSizeId) {
+            $connection->executeStatement('
+                    REPLACE INTO `media_folder_configuration_media_thumbnail_size` (`media_folder_configuration_id`, `media_thumbnail_size_id`)
+                    VALUES (:folderConfigurationId, :thumbnailSizeId)
+                ', [
+                'folderConfigurationId' => $configurationId,
+                'thumbnailSizeId' => $thumbnailSizeId,
+            ]);
+        }
+
+        $this->registerIndexer($connection, 'media_folder_configuration.indexer');
+    }
+
+    public function updateDestructive(Connection $connection): void
+    {
+        // implement update destructive
+    }
+
+    /**
+     * @return array<string>
+     */
+    private function upsertThumbnailSizes(Connection $connection): array
```

Comment:
Extract this into a Trait in `Traits` folder and them call them :). Both migrations do basically the same just different sizes and media folder